datalad/runner/exception.py

```diff
@@ -35,17 +35,14 @@ def __init__(self, cmd="", msg="", code=None, stdout="", stderr="", cwd=None,
         self.kwargs = kwargs
 
     def to_str(self, include_output=True):
-        from datalad.utils import (
-            ensure_unicode,
-            join_cmdline,
-        )
+        from datalad.utils import ensure_unicode
         to_str = "{}: ".format(self.__class__.__name__)
         cmd = self.cmd
         if cmd:
             to_str += "'{}'".format(
                 # go for a compact, normal looking, properly quoted
                 # command rendering if the command is in list form
-                join_cmdline(cmd) if isinstance(cmd, list) else cmd
+                " ".join(cmd) if isinstance(cmd, list) else cmd
```

Comment:
why to not use `join_cmdline` here to ensure that command could be just copy/pasted to be ran?

Reply:
Thx! Can't remember why that was changed, reverted the change

Reply:
Thx for catching that. Not sure why I changed it, but it is restored to usage of `join_cmdline`.